packages/jellyfish-api-core/src/category/masternode.ts

```diff
@@ -175,6 +175,20 @@ export class Masternode {
     return await this.client.call('setgovheight', [input, activationHeight, utxos], 'number')
   }
 
+  /**
+   * Unset special governance variables
+   *
+   * @param {Record<string, number | string | string[]>} input json object
+   * @param {UTXO[]} [utxos = []] Specific utxos to spend
+   * @param {string} [utxos.txid] The transaction id
+   * @param {string} [utxos.vout] The output number
+   * @return {Promise<string>} hash
+   *
+   */
+  async unsetGov (input: Record<string, number | string | string[]>, utxos: UTXO[] = []): Promise<string> {
```

Comment:
`nit`: lets call 'input', 'variables' instead, following what's used in the defichain rpc, which is more descriptive.

```suggestion
  async unsetGov (variables: Record<string, number | string | string[]>, utxos: UTXO[] = []): Promise<string> {
```

Reply:
Other than this should be good!